### DISCUSSION TITLE
Feature: add funktion to Connect without Password

### DISCUSSION THREAD

**Discussion by xBr0th3rx** [rustdesk/rustdesk-server-pro#474] [Feature Request]
Hello everyone,

we have Rustdesk Pro for our company and need the function that we can connect without a password when no one is logged in yet, i.e. in the Linux login screen.

The whole thing should be able to be set in the server, e.g. only group administrators are allowed to do this.

**Comment by SerWax** [comment]
You can preset a password in the custom client and use a strategy that allows permanent password.
No new feature required.

If you want no password at all, that is not a good idea.
